contracts/contracts/RequestManager.sol

```diff
@@ -55,12 +55,13 @@ contract RequestManager is Ownable, LpWhitelist {
     ///
     /// .. seealso:: :sol:func:`createRequest`
     event RequestCreated(
-        uint256 requestId,
+        bytes32 requestId,
```

Comment:
We should probably make the request ID `indexed`.

Reply:
good point